app/src/main/java/com/gh4a/utils/HttpImageGetter.java

```diff
@@ -440,14 +436,13 @@ private Drawable loadImageForUrl(String source) {
         Bitmap bitmap = null;
 
         if (!mDestroyed && url != null) {
-            File output = null;
-            InputStream is = null;
             Request request = new Request.Builder()
                     .url(url)
                     .build();
             try (Response response = mClient.newCall(request).execute()) {
-                is = response.body().byteStream();
-                if (is != null) {
+                if (response.body() != null) {
+                    byte[] responseBody = response.body().bytes();
+                    InputStream is = new ByteArrayInputStream(responseBody);
```

Comment:
An interesting sidenote: using a `ByteArrayInputStream` here instead of OkHttp `byteStream` makes `URLConnection.guessContentTypeFromStream` (used at line 452) actually work, since OkHttp streams do not support marking which is required by the latter function to work.

Reply:
That fact probably deserves a comment.